xarray/test/test_dataarray.py

```diff
@@ -1549,20 +1549,31 @@ def test_rolling_wrapped_bottleneck(self):
 
         # Using center=False
         rolling_obj = da.rolling(time=7, center=False)
-        for name in ('sum', 'mean', 'std', 'min', 'max', 'median'):
+        for name in ('sum', 'mean', 'std', 'min', 'max', 'median', 'count'):
             actual = getattr(rolling_obj, name)()['time']
             self.assertDataArrayEqual(actual, da['time'])
 
         # Using center=True
         rolling_obj = da.rolling(time=7, center=True)
-        for name in ('sum', 'mean', 'std', 'min', 'max', 'median'):
+        for name in ('sum', 'mean', 'std', 'min', 'max', 'median', 'count'):
             actual = getattr(rolling_obj, name)()['time']
             self.assertDataArrayEqual(actual, da['time'])
 
         # catching invalid args
         with self.assertRaisesRegexp(ValueError, 'Rolling.median does not'):
             da.rolling(time=7, min_periods=1).median()
 
+    def test_rolling_count_correct(self):
+
+        da = self.make_rolling_example_array()
+        result = da.rolling(time=7, min_periods=1).count()
+        expected_rolling_count = np.r_[np.arange(1, 8), np.full((14,), 7)]
+        expected = xr.DataArray(
+            np.resize(expected_rolling_count, (4, 21)).T,
```

Comment:
Use `xarray.core.npcompat.broadcast_to`, which is just a back port of `np.broadcast_to`.

I'm honestly not sure how `np.resize` does broadcasting. It's a strange function -- very different from the low-level `resize` method.